projects/Apigen/Apigen.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

When targeting a single framework, it's best to switch back to `TargetFramework`

```suggestion
    <TargetFramework>net6.0</TargetFramework>
```